R/summary_functions.R

```diff
@@ -4,8 +4,9 @@
 #' @export
 percent_coverage <- function(coverage_result){
   coverage_data <- as.data.frame(coverage_result)
-  num_lines <- nrow(coverage_data)
-  num_not_zero <- sum(coverage_data$value != 0) # how many lines have a value that is not zero?
+  num_lines <- length(unique(coverage_data$first_line))
```

Comment:
Hmm I see. But the problem is that when you visually look at the results in the source files, if on the same line you have one expression that is covered and one that is not, how do you represent it ? 
In the same spirit, the developer would look at the missed lines, and it will be hard to grasp why a line is missed although he can be sure that the main expression on this line was executed, do you see what I mean ?